### PR TITLE
Update terminology frontend boilerplate docs - Closes 316 in lisk-sdk-examples

### LINKED ISSUE #316: Update Lisk chat links
As we moved for all our chat channels to Discord now, we need to update all links in the documentation to point to the right Discord channels.

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #316 in lisk-sdk-examples

### CODE REVIEW

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 1]
```diff
@@ -726,11 +726,11 @@ Minimum and maximum values for the certificate threshold are calculated using th
...
 (async () => {
-	// Update the path to point to your sidechain client data folder
-	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
-	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
+	// Update the path to point to your blockchain client data folder
+	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
+	const sidechainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
 	// Get active validators from sidechain
-	const bftParams = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
+	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]: Here, the sidechain client is meant in the context. Please revert the changes.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]: Ummmm, the client can only be a blockchain client per the terminology agreement. A node can be for a sidechain or a mainchain, however, the client can only be a blockchain client. I can update the comment to say something like this:
"// Update the path to point to your blockchain client data folder for the sidechain" to make it clearer.

Comment (chris529) [reply docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]: Yes, that sounds good..I think to further enhance the clarity in this statement, imho I would suggest the following:
"// Update the path to point to your blockchain client data folder for the respective sidechain"

**Review by chris529 — COMMENTED** [review]
Looks great! Only 1 minor suggestion 👍

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 2]
```diff
@@ -726,11 +726,11 @@ Minimum and maximum values for the certificate threshold are calculated using th
 const { apiClient } = require('@liskhq/lisk-client');
 
 (async () => {
-	// Update the path to point to your sidechain client data folder
-	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
-	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
+	// Update the path to point to your blockchain client data folder
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
	// Update the path to point to your blockchain client data folder for the respective sidechain
```

**Review by chris529 — APPROVED** [review]
LGTM! 👍

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc (hunk 1 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]: I think this is unnecessary complicated. In this specific context of sidechain/mainchain communication, it still makes sense to talk about sidechain and mainchain nodes and blockchain clients. Also, if you want to be 100% precise here, you are connecting to a node, not the client. In the end, we would also need to update this in the code and I don't think it is necessary. Please just revert the changes.

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 3]
```diff
@@ -726,11 +726,11 @@ Minimum and maximum values for the certificate threshold are calculated using th
 const { apiClient } = require('@liskhq/lisk-client');
 
 (async () => {
-	// Update the path to point to your sidechain client data folder
-	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
-	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
+	// Update the path to point to your blockchain client data folder for the respective sidechain
+	const blockchainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
+	const sidechainNodeInfo = await blockchainClient.invoke('system_getNodeInfo');
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
	// Update the path to point to your sidechain client data folder
	const sidechainClient = await apiClient.createIPCClient('~/.lisk/hello_client');
	const sidechainNodeInfo = await sidechainClient.invoke('system_getNodeInfo');
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc [hunk 4]
```diff
@@ -726,11 +726,11 @@ Minimum and maximum values for the certificate threshold are calculated using th
...
 	// Get active validators from sidechain
-	const bftParams = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
+	const bftParams = await blockchainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]:
```suggestion
	const bftParams = await sidechainClient.invoke('consensus_getBFTParameters', { height: sidechainNodeInfo.height });
```

File: docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/build-blockchain/register-sidechain.adoc]: Changes reverted.

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**